homework/App.js

```diff
@@ -15,14 +15,70 @@ class App {
     // Add code here to initialize your app
     // 1. Create the fixed HTML elements of your page
     // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
-
     const root = document.getElementById('root');
-
-    Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
+    const header = Util.createAndAppend('header', root, { id: 'header' });
+    Util.createAndAppend('p', header, { text: 'HYF Repositories' });
+    const select = Util.createAndAppend('select', header, { id: 'select' });
+    const divGeneral = Util.createAndAppend('div', root, { id: 'div_general' });
+    const leftPanel = Util.createAndAppend('div', divGeneral, { id: 'left_panel' });
+    const leftPanelContainer = Util.createAndAppend('div', leftPanel);
+    const leftPanelContainerTable = Util.createAndAppend('table', leftPanelContainer);
+    const repositoryProperties = ['Repository :', 'Description :', 'Forks :', 'Updated :'];
+    const leftPanelContainerTableFirstTr = Util.createAndAppend('tr', leftPanelContainerTable);
+    Util.createAndAppend('td', leftPanelContainerTableFirstTr, {
+      text: repositoryProperties[0],
+    });
+    const linkRepo = Util.createAndAppend('td', leftPanelContainerTableFirstTr);
+    Util.createAndAppend('a', linkRepo, { id: 'property_repository' });
+    const rightPanel = Util.createAndAppend('div', divGeneral, { id: 'right_panel' });
+    Util.createAndAppend('p', rightPanel, { text: 'Contributions', id: 'right_column_title' });
+    const rigthColumnListDefault = Util.createAndAppend('ul', rightPanel, {
+      id: 'right_column_list',
+    });
 
     try {
-      const repos = await Util.fetchJSON(url);
-      this.repos = repos.map(repo => new Repository(repo));
+      const data = await Util.fetchJSON(url);
+      this.data = data.map(repo => new Repository(repo));
+      this.data = data.sort((a, b) => a.name.localeCompare(b.name));
+      const contributorsURL = this.data.map(item => item.contributors_url);
+      const fetchedDataContributorsDefaultRaw = await fetch(contributorsURL[0]);
```

Comment:
Why not use `Utils.fetchJSON` here as well. 😉

Reply:
Replaced.